### ISSUE TITLE
Inconsistent data passed to toolltip when using keyboard nav

### ISSUE THREAD

**Issue by cvkline** [emeeks/semiotic#259]
With an OrdinalFrame pie chart using `hoverAnnotation={true}` and a `tooltipContent` function, the object passed to the function when hovering with the mouse is a `type='column-hover'` as we would expect, and contains a `column` object with percentages and piece data. (first attachment)

However when keyboard nav is used to navigate through the hovers in an accessible way, the object passed to the function is a `type='frame-hover'` which does not have the `column` object inside it. (second attachment).

This can be worked around with conditionals, but I would expect the same data in the tooltip no matter how it is activated.

![screen shot 2018-06-12 at 4 31 44 pm](https://user-images.githubusercontent.com/6576592/41318939-b7fd8e68-6e5f-11e8-95ca-0fdd7789567d.png)

![screen shot 2018-06-12 at 4 38 24 pm](https://user-images.githubusercontent.com/6576592/41318945-bc9275a6-6e5f-11e8-96ff-cb04144c2cac.png)

**Comment by mmazanec22** [comment]
Ditto to this with NetworkFrame
<img width="661" alt="screen shot showing console logs of value passed to tooltipContent" src="https://user-images.githubusercontent.com/12260067/41359484-994ef670-6ef8-11e8-9564-e3f90438b9f0.png">

**Comment by emeeks** [comment]
Should be fixed in 1.11.7.

**Comment by mmazanec22** [comment]
@cvkline is this because you are using column hover for the tooltip rather than piece hover, but piece hover is the default behavior of keyboard nav?

**Comment by cvkline** [comment]
I'm just using `hoverAnnotation={true}` and then providing a `toolTipContent` function.

**Comment by emeeks** [comment]
Alright I'll reopen this since and dive into it with the other accessibility issues.

**Comment by daschi** [comment]
Hello! I have a similar issue with the `ResponsiveXYFrame` using `hoverAnnotation={true}` and `tooltipContent`, it seems that on keyboard navigation with voiceover, the voiceover describes the chart as navigable with left/right arrows for the tooltip, but then when hitting the keys the tooltip is not receiving the `frame-hover` object. I have 2 lines on this chart using the prop `lines`, and after tabbing 3x I am able to navigate to the tooltips with the left/right arrows. I'm not super familiar with ARIA but it seems like maybe the description to navigate is on the wrong element? Let me know if I should add any additional details or steps to reproduce. Thank you so much!!
